apps/remix-ide-e2e/src/tests/solidityUnittests.spec.ts

```diff
@@ -90,18 +90,21 @@ module.exports = {
       .waitForElementContainsText('*[data-id="testTabTestsExecutionStopped"]', 'The test execution has been stopped', 60000)
   },
 
-  'Should fail on compilation': function (browser: NightwatchBrowser) {
+  'Should fail on compilation, open file on error click, not disappear error': function (browser: NightwatchBrowser) {
```

Comment:
not sure what you mean by `not disappear error` ?

Reply:
By default in SUT plugin, a file is opened, it clears the result. So if error appears in a file which is not there in editor, on clicking on error, it will open the file and consequently, compilation error will be cleared